catalog/app/containers/Notifications/index.js

```diff
@@ -48,19 +43,18 @@ export const Display = composeComponent(
     )),
 )
 
-export const WithNotifications = composeComponent(
-  'Notifications.WithNotifications',
-  ({ children }) => (
-    <React.Fragment>
+export function WithNotifications({ children }) {
```

Comment:
what's the reason to move away from arrow functions?

Reply:
to have useful stack traces (anonymous functions are rendered as `Unknown`)